lib/Doctrine/DBAL/Connections/MasterSlaveConnection.php

```diff
@@ -120,7 +120,9 @@ public function isConnectedToMaster()
     }
 
     /**
-     * {@inheritDoc}
```

Comment:
It seems like PHPStan doesn't accept adding info together with `{@inheritDoc}`. Is that correct @ondrejmirtes?

Reply:
Yes, it doesn’t merge that. Either you are inheriting the phpDoc, or overriding it.

Reply:
FWIW, the `@return` annotation [should be inherited](https://docs.phpdoc.org/guides/inheritance.html#methods) according to the spec regardless of `{@inheritDoc}` that is used to [import the description of a parent element](https://docs.phpdoc.org/guides/inheritance.html#the-inheritdoc-tag).